ci/prow/config.yaml

```diff
@@ -1739,6 +1753,23 @@ periodics:
       - "./hack/release.sh"
       - "--publish"
       - "--tag-release"
+- cron: "15 9 * * *" # Run at 02:15PST every day (09:15 UTC)
+  name: ci-knative-backup-images
+  agent: kubernetes
+  labels:
+    preset-backup-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/go-containerregistry/gcrane:latest
```

Comment:
This config is missing setting up the service account for gcrane to use. Is there a flag you can add to point to `/etc/backup-account/service-account.json`? Otherwise, would a `gcloud auth activate-service-account --key-file=/etc/backup-account/service-account.json` work?

Reply:
`gcrane` uses the oauth2 library's [`google.FindDefaultCredentials`](https://godoc.org/golang.org/x/oauth2/google#FindDefaultCredentials) to look for credentials, see [here](https://github.com/google/go-containerregistry/blob/e0f538712eb0767ae6959531febe6041fa466602/pkg/v1/google/auth.go#L49).

I _believe_ that should be sufficient, since it looks like we set `GOOGLE_APPLICATION_CREDENTIALS` in the environment setup (line 138), does that make sense or am I misreading this config?

Reply:
Sweet, problem solved neatly.